diesel/src/util.rs

```diff
@@ -0,0 +1,5 @@
+pub trait TupleAppend<T> {
```

Comment:
Can you give this a short doc comment mentioning the purpose (in contrast to AppendSelection) and the impls in `impls/tuples.rs`?

I would probably like to add some tests (aside from the integration tests) but I'm not sure where best to put that and how to only test this without writing a whole bunch of impls… so, yeah, I'm good with not testing this right now 😅